MC/run/ANCHOR/anchorMC.sh

```diff
@@ -0,0 +1,181 @@
+if [ "${NO_MC}" ]; then
+  return ${RECO_RC} 2>/dev/null || exit ${RECO_RC} # optionally quit here and don't do MC (useful for testing)
+fi
+
+ALIEN_JDL_LPMPRODUCTIONTAG=$ALIEN_JDL_LPMPRODUCTIONTAG_KEEP
+echo "Setting back ALIEN_JDL_LPMPRODUCTIONTAG to $ALIEN_JDL_LPMPRODUCTIONTAG"
+
+# now create the local MC config file --> config-config.json
+${O2DPG_ROOT}/UTILS/parse-async-WorkflowConfig.py
+
+# check if config reasonably created
+if [[ `grep "o2-ctf-reader-workflow-options" config-json.json 2> /dev/null | wc -l` == "0" ]]; then
+  echo "Problem in anchor config creation. Stopping."
+  exit 1
+fi
+
+# -- CREATE THE MC JOB DESCRIPTION ANCHORED TO RUN --
+
+NWORKERS=${NWORKERS:-8}
+MODULES="--skipModules ZDC"
+SIMENGINE=${SIMENGINE:-TGeant4}
+SIMENGINE=${ALIEN_JDL_SIMENGINE:-${SIMENGINE}}
+NTIMEFRAMES=${NTIMEFRAMES:-50}
+NSIGEVENTS=${NSIGEVENTS:-22}
+
+SPLITID=${SPLITID:-0}
+PRODSPLIT=${PRODSPLIT:-100}
+CYCLE=${CYCLE:-0}
+# let SEED=$SPLITID+$CYCLE*$PRODSPLIT
+SEED=${ALIEN_PROC_ID}
+
+# create workflow
+# THIS NEEDS TO COME FROM OUTSIDE
+# echo "$" | awk -F' -- ' '{print $1, $3}'
+
+#baseargs="-col PbPb -eCM 5360 -tf ${NTIMEFRAMES} --split-id ${SPLITID} --prod-split ${PRODSPLIT} --cycle ${CYCLE} --run-number ${RUNNUMBER}"
+baseargs="-tf ${NTIMEFRAMES} --split-id ${SPLITID} --prod-split ${PRODSPLIT} --cycle ${CYCLE} --run-number ${RUNNUMBER}"
+
+# THIS NEEDS TO COME FROM OUTSIDE
+remainingargs="-gen pythia8 -proc heavy_ion -seed ${SEED} -ns ${NSIGEVENTS} --include-local-qc --pregenCollContext"
+
+remainingargs="${remainingargs} -e ${SIMENGINE} -j ${NWORKERS}"
+remainingargs="${remainingargs} -productionTag ${ALIEN_JDL_LPMPRODUCTIONTAG:-alibi_anchorTest_tmp}"
+remainingargs="${remainingargs} --anchor-config config-json.json"
+
+echo "baseargs: ${baseargs}"
+echo "remainingargs: ${remainingargs}"
+              
+# query CCDB has changed, w/o "_"
+${O2DPG_ROOT}/MC/bin/o2dpg_sim_workflow_anchored.py ${baseargs} -- ${remainingargs} &> timestampsampling_${RUNNUMBER}.log
+[ "$?" != "0" ] && echo "Problem during anchor timestamp sampling " && exit 1
+
+TIMESTAMP=`grep "Determined timestamp to be" timestampsampling_${RUNNUMBER}.log | awk '//{print $6}'`
+echo "TIMESTAMP IS ${TIMESTAMP}"
+
+# -- PREFETCH CCDB OBJECTS TO DISC      --
+# (make sure the right objects at the right timestamp are fetched
+#  until https://alice.its.cern.ch/jira/browse/O2-2852 is fixed)
+export ALICEO2_CCDB_LOCALCACHE=$PWD/.ccdb
+[ ! -d .ccdb ] && mkdir .ccdb
+
+CCDBOBJECTS="/CTP/Calib/OrbitReset /GLO/Config/GRPMagField/ /GLO/Config/GRPLHCIF /ITS/Calib/DeadMap /ITS/Calib/NoiseMap /ITS/Calib/ClusterDictionary /TPC/Calib/PadGainFull /TPC/Calib/TopologyGain /TPC/Calib/TimeGain /TPC/Calib/PadGainResidual /TPC/Config/FEEPad /TOF/Calib/Diagnostic /TOF/Calib/LHCphase /TOF/Calib/FEELIGHT /TOF/Calib/ChannelCalib /MFT/Calib/DeadMap /MFT/Calib/NoiseMap /MFT/Calib/ClusterDictionary /FT0/Calibration/ChannelTimeOffset /FV0/Calibration/ChannelTimeOffset /GLO/GRP/BunchFilling"
```

Comment:
Hello @sawenzel , @catalinristea , 
Do we still need these?

Reply:
Note also that the list contains non-existing objects, 
```
[ERROR] Requested resource does not exist: http://alice-ccdb.cern.ch///FT0/Calibration/ChannelTimeOffset/1686209727558/
[ERROR] Unable to find object /FT0/Calibration/ChannelTimeOffset/1686209727558, Aborting
[ERROR] Requested resource does not exist: http://alice-ccdb.cern.ch///FV0/Calibration/ChannelTimeOffset/1686209727558/
[ERROR] Unable to find object /FV0/Calibration/ChannelTimeOffset/1686209727558, Aborting
[ERROR] Requested resource does not exist: http://alice-ccdb.cern.ch///GLO/GRP/BunchFilling/1686209727558/
[ERROR] Unable to find object /GLO/GRP/BunchFilling/1686209727558, Aborting
```

Reply:
These are no longer needed.

Reply:
Done, thanks!